app/views/home/federation_portals_status.html.haml

```diff
@@ -0,0 +1,5 @@
+= render TurboFrameComponent.new(id:"federation_portals_status_#{@key}") do |container|
```

Comment:
no need for this file you can render it directly in the controller, 

soothing like this 

```ruby 
render inline: helpers.your_helper
```

one less file is always a win, future us.